packages/dds/tree/src/feature-libraries/modular-schema/fieldChangeHandler.ts

```diff
@@ -74,6 +71,7 @@ export interface FieldChangeRebaser<TChangeset> {
 		invertedChange: TChangeset,
 		originalRevision: RevisionTag | undefined,
 		genId: IdAllocator,
+		// reviver: NodeReviver,
```

Comment:
While I'm guessing none of our FieldKind implementations need the reviver during the amend phase, I would include it for the sake of generality (it's concievable that some FieldKind would need it).